packages/checkout-extension/src/handler/queryHandlers/createGetConsignmentHandler.ts

```diff
@@ -1,14 +1,18 @@
-import { ExtensionCommandType, ExtensionMessageType } from '@bigcommerce/checkout-sdk';
+import { ExtensionMessageType, ExtensionQueryType } from '@bigcommerce/checkout-sdk';
 
-import { CommandHandler, HandlerProps } from './CommandHandler';
+import { QueryHandler, QueryHandlerProps } from './QueryHandler';
 
 export function createGetConsignmentHandler({
     checkoutService,
     extension,
-}: HandlerProps): CommandHandler<ExtensionCommandType.GetConsignments> {
+}: QueryHandlerProps): QueryHandler<ExtensionQueryType.GetConsignments> {
     return {
-        commandType: ExtensionCommandType.GetConsignments,
-        handler: () => {
+        queryType: ExtensionQueryType.GetConsignments,
+        handler: async (data) => {
+            if (!data.payload?.useCache) {
+                await checkoutService.loadCheckout();
```

Comment:
Could this cause side effects that might change the display of values on checkout page?

Reply:
Yes, it will refresh the checkout state, hence updating the checkout page UI accordingly.
I feel it is in line with our existing logic, checkout page should be in sync with the remote data after an action.